src/openforms/contrib/kadaster/api/views.py

```diff
@@ -52,6 +52,8 @@ class AddressAutocompleteView(APIView):
             "Get the street name and city for a given postal code and house number.\n\n"
             "**NOTE** the `/api/v2/location/get-street-name-and-city/` endpoint will "
             "be removed in v3. Use `/api/v2/geo/address-autocomplete/` instead."
+            "Deriving the city and street name from the texboxes is deprecated and will be removed"
+            "in a future version. Instead, the addressNL component should be used."
```

Comment:
I think this can be removed - the API endpoint doesn't necessarily change, right? It just returns a little bit of extra data.